doc/source/reference/api/api.rst

```diff
@@ -734,7 +734,7 @@ particularly with complicated layouts.
    ~yt.visualization.eps_writer.single_plot
    ~yt.visualization.eps_writer.multiplot
    ~yt.visualization.eps_writer.multiplot_yt
-   ~yt.visualization.eps_writer.return_cmap
+   ~yt.visualization.eps_writer.return_colormap
```

Comment:
I would keep the old references here too so it's clear they are deprecated functions if somebody uses them and comes across the API docs. Also, the references to the minorticks will automatically include both the deprecated and updated functions so this way we're being consistent.

Reply:
done :)